FirebasePerformance/Sources/Timer/FIRTrace.m

```diff
@@ -139,9 +142,9 @@ - (void)start {
     self.backgroundActivityTracker = [[FPRTraceBackgroundActivityTracker alloc] init];
     FPRSessionManager *sessionManager = [FPRSessionManager sharedInstance];
     if (!self.isStage) {
-      [self updateTraceWithSessionId];
+      [self updateTraceWithSessionId:[[sessionManager sessionDetails] copy]];
```

Comment:
nit: Dot syntax should be used to access a property:
```
[sessionManager.sessionDetails copy]
```

Reply:
Done.